tests/integration/init/schemas/test_init_with_schemas_command.py

```diff
@@ -220,6 +220,37 @@ def test_init_interactive_with_event_bridge_app_aws_schemas_python(self):
             self.assertTrue(expected_output_folder.is_dir())
             self.assertTrue(Path(expected_output_folder, "hello_world_function", "schema").is_dir())
 
+    def test_init_interactive_with_event_bridge_app_aws_schemas_go(self):
```

Comment:
please add another use case to test the `event bridge schema app` template, and mark it `@skipif(SKIP_SCHEMA_TESTS)`

Reply:
sorry I got confused, so tag this one with `@skipif(SKIP_SCHEMA_TESTS)`

Reply:
This entire test class is already being skipped by this line: https://github.com/aws/aws-sam-cli/blob/develop/tests/integration/init/schemas/test_init_with_schemas_command.py#L17